File: app/src/main/java/com/gamalinda/android/poc/testing/collapsingtoolbar/fragment/TextContentMockFragment.java
package com.gamalinda.android.poc.testing.collapsingtoolbar.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.gamalinda.android.poc.testing.collapsingtoolbar.R;

public class TextContentMockFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View rootView = inflater.inflate(
                R.layout.content_main, container, false);
        return rootView;
    }
}

File: app/src/main/java/com/gamalinda/android/poc/testing/collapsingtoolbar/fragment/TextContentMockFragment.kt
package com.gamalinda.android.poc.testing.collapsingtoolbar.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

import com.gamalinda.android.poc.testing.collapsingtoolbar.R

class TextContentMockFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(
                R.layout.content_main, container, false)
    }
}
